src/test/java/com/sforce/cd/apexUnit/fileReader/ApexManifestFileReaderTest.java

```diff
@@ -155,4 +157,20 @@ public void cleanUpTestFiles() {
 			LOG.info("Test files directory does not exist; hence not deleted");
 		}
 	}
+
+	@Test
+	public void namespaceParserTest() {
```

Comment:
👍  
Few more - 
1. limitations on length of namespace?
2. special characters?